### PR TITLE
[CRM-4]: Create, run services + Eureka registry

### PR DESCRIPTION
**Pull request by momopoi443**
Created Spring Boot projects for all services and Eureka discovery service. Configured application.properties so services can register in Eureka server, also configured setting for server. Manually tested: services successfully registered.

### CODE REVIEW

File: payment-service/pom.xml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>3.1.2</version>
+		<relativePath/> <!-- lookup parent from repository -->
+	</parent>
+	<groupId>com.crm-food-establishment</groupId>
+	<artifactId>payment-service</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<name>payment-service</name>
```
Comment (polinaucc) [inline payment-service/pom.xml:14]: Let's add here meaningful description

File: user-managemer-service/pom.xml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>3.1.2</version>
+		<relativePath/> <!-- lookup parent from repository -->
+	</parent>
+	<groupId>com.crm-food-establishment</groupId>
+	<artifactId>user-manager</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<name>user-manager</name>
+	<description>Demo project for Spring Boot</description>
```
Comment (polinaucc) [inline user-managemer-service/pom.xml:15]: Let's change the description here

File: user-manager-service/src/main/resources/db/changelog/1.0-SNAPSHOT/user-manager_init_schemas.xml [hunk 3]
```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <changeSet id="1" author="ygoway">
+        <createTable tableName="user">
```
Comment (polinaucc) [inline user-manager-service/src/main/resources/db/changelog/1.0-SNAPSHOT/user-manager_init_schemas.xml:7]: Let's add 'comment' here and let's add spaces (enter) between changesets

File: user-manager-service/src/main/resources/db/changelog/db.changelog-master.yaml [hunk 4]
```diff
@@ -0,0 +1,3 @@
+databaseChangeLog:
```
Comment (polinaucc) [inline user-manager-service/src/main/resources/db/changelog/db.changelog-master.yaml:1]: Let's keep xml better

File: user-manager-service/src/main/resources/db/changelog/db.changelog-master.yaml [hunk 5]
```diff
@@ -0,0 +1,3 @@
...
+  - include:
+      file: db/changelog/1.0-SNAPSHOT/user-manager_init_schemas.xml
```
Comment (polinaucc) [inline user-manager-service/src/main/resources/db/changelog/db.changelog-master.yaml:3]: Let's remoe 1.0-SNAPSHOT directory

**Review by ygoway — APPROVED** [review]

**Review by polinaucc — APPROVED** [review]

**Pull request merged**